client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/SutInfoDto.java

```diff
@@ -49,7 +49,15 @@ public enum OutputFormat {
         KOTLIN_JUNIT_4,
         KOTLIN_JUNIT_5,
         JS_JEST,
-        CSHARP_XUNIT
+        CSHARP_XUNIT;
+
+        public boolean isJava(){
+            return this.name().startsWith("JAVA");
+        }
+
+        public boolean isJavaOrKotlin(){
+            return this.name().startsWith("JAVA") || this.name().startsWith("KOTLIN");
+        }
```

Comment:
2 issues here:

1) to avoid code duplication, `isJavaOrKotlin()` should call `isJava()`

2) should not have business logic in the DTOs... rather create a util class outside of `dto` package (eg `DtoUtils`). might sound convoluted, but keep in mind that we might in the future delete the `dto` package and rather create automatically from an OpenAPI schema...